list/PageableList.js

```diff
@@ -315,8 +221,87 @@ define([
 			};
 		}),
 
+		/*jshint maxcomplexity:14*/
+		computeProperties: function (props) {
+			if (this.pageLength > 0) {
+				if ("_busy" in props || "hideOnPageLoad" in props || "autoPaging" in props || "showNoItems" in props) {
+					this._updateListView();
+				}
+
+				if ("loadPreviousMessage" in props || "loadingMessage" in props || "_showPreviousPageLoader" in props) {
+					if (this.previousPageLoader && this.previousPageLoader.deliver) {
+						this.previousPageLoader.labels = {
+							loadMessage: string.substitute(this.loadPreviousMessage, this),
+							loadingMessage: this.loadingMessage
+						};
+						this.previousPageLoader.deliver();
```

Comment:
`string.substitute(this.loadPreviousMessage, this)` means that loadMessage could change as a response to any property change at all.

So I think probably you should move this logic outside of all the `if()` statements and just say something like:

```
this.previousPageLoader.label = this.previousPageLoader.loading ?
   this.loadingMessage : string.substitute(this.loadPreviousMessage, this);
```